robot_ws/src/hello_world_robot/package.xml

```diff
@@ -1,27 +1,21 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
   <name>hello_world_robot</name>
-  <version>1.2.0</version>
+  <version>1.0.0</version>
```

Comment:
Do you think we should reset the version numbers in the ROS2 versions of our CEs/SAs? I don't feel strongly one way or the other, we just need to make a decision and be consistent about it. Not resetting the versions might have an advantage from a release automation perspective though.

Reply:
Maybe version is 2.0 for ros2?

Reply:
Great question here, is it possible to have a lexicographical version? Such as ros2-1.0.0?